### ISSUE TITLE
Octave compatibility: 'feature' does not exist in Octave

### ISSUE THREAD

**Issue by kupiqu** [DynaSim/DynaSim#223]
@jsherfey how do you think it is better to deal with this?

```
error: 'feature' undefined near line 306 column 29
error: called from
    dsSimulate at line 306 column 1
    dsDemos at line 136 column 5
```

**Comment by kupiqu** [comment]
We could use matlab specific features only in matlab by using this function:

```
function ui = reportUI
% function that reports ui, that is whether Octave or matlab is in use
LIC = license('inuse');
ui = LIC.feature;
% matlab toolboxes and octave packages are added to inuse, unfortunately 'matlab'' appears last (after tootlboxes) and octave first (before packages), what follows is a workaround to always report the proper UI:
if ~strcmp(ui,'octave')
  ui = 'matlab';
end
```

**Comment by kupiqu** [comment]
fixed in commit 833c5fc

**Comment by kupiqu** [comment]
adding some more info for octave in commit 9bd7a2f
